docs/releases/3.0.txt

```diff
@@ -185,7 +185,13 @@ Templates
 Tests
 ~~~~~
 
-* ...
+* The new test :class:`~django.test.Client` argument
```

Comment:
Normally I keep the release note simpler (1 sentence) and link to the documentation that provides a more information. In this case the release note seems more detailed than the documentation which is backwards.

Also, I think the documentation could better describe the use case.